nmdc.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python
```

Comment:
Then we must edit the readme.txt
On Mar 27, 2015 4:12 AM, "Bharadwaj Machiraju" <notifications@github.com>
wrote:

> In nmdc.py
> <https://github.com/tunnelshade/dcfury/pull/1#discussion_r27263397>:
>
> > @@ -1,3 +1,5 @@
> > +#!/usr/bin/python
>
> The problem with this line is that I use homebrew on OSX. So, my python is
> at /usr/local/bin/python2.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tunnelshade/dcfury/pull/1/files#r27263397>.
>